skyrim-platform/src/platform_se/codegen/TSConverter.cpp

```diff
@@ -86,48 +86,59 @@ int main(int argc, char* argv[])
 
   auto parseReturnValue = [&](std::string rawType,
                               std::string objectTypeName) -> std::string {
-    if (rawType == "Int" || rawType == "Float") {
```

Comment:
Please could you bring back the version with `if`. This one seems incorrect

Reply:
ready